app/src/test/java/org/simple/clinic/summary/PatientSummaryUpdateTest.kt

```diff
@@ -189,9 +208,14 @@ class PatientSummaryUpdateTest {
   }
 
   @Test
-  fun `when at least one measurement is present and diagnosis is not recorded, clicking on save must show diagnosis error`() {
+  fun `when at least one measurement is present and diagnosis is not recorded and diabetes management is enabled, clicking on save must show diagnosis error`() {
```

Comment:
Can we add a corresponding test to verify the behaviour is unchanged when the diabetes management flag is disabled?